jhucourse/woj/backend/game.py

```diff
@@ -0,0 +1,134 @@
+'import json'
+import numpy
+import player
+'import QuestionBank'
+
+class game:
+    def __init__(self):
+        self.player_list = []
+        self.current_player = 0
```

Comment:
This is going to be a string so I would just set this to an empty string. self.current_player = ""

Reply:
In the design document it was labeled as an integer to refer to a spot in the array. How would it work as a string?